### PR TITLE
Fix errors when re-ordering using dnd

### LINKED ISSUE #1780: Elements disappear after drag-drop and app becomes undeletable.
## Current Behavior

Elements disappear from the builder after rearranging them using drag and drop. 
Also the app becomes undeletable. To delete the app, the broken page should be deleted first then the app.

## Expected Behavior

Rearranging elements using dnd should not cause any issues.

## Steps to Reproduce

https://user-images.githubusercontent.com/48331678/188312044-5c00af10-2eed-4dbd-9aae-1f1e918838fe.mov

### PR DESCRIPTION
**Pull request by melhamin**
This PR fixes errors while re-ordering elements using dnd.

## Related Issue(s)

Fixes #1780

### CODE REVIEW

File: libs/frontend/modules/element/src/store/element.service.ts [hunk 1]
```diff
@@ -302,51 +300,31 @@ parent
...
+      // Detach from prev sibling
       element.makeDetachPrevSiblingInput(),
     ]
 
-    updateElementCacheFns.push(
+    const updateElementCacheFns: Array<() => void> = [
+      // Detach from parent
       element.detachParent(),
+      // Attach next to prev
+      element.attachPrevToNextSibling(),
```
Comment (webberwang) [inline libs/frontend/modules/element/src/store/element.service.ts:315]: For all these, if we used `element.attachPrevToNextSibling` and removed the `() => {}` inside the method, would that work?

**Review by webberwang — CHANGES_REQUESTED** [review]

File: libs/frontend/modules/element/src/store/element.service.ts (hunk 1 above)
Comment (melhamin) [reply libs/frontend/modules/element/src/store/element.service.ts:315]: I believe it should work.

Comment (melhamin) [reply libs/frontend/modules/element/src/store/element.service.ts:315]: No sorry it doesn't work

**Review by webberwang — CHANGES_REQUESTED** [review]
Could remove the custom element endpoint for the batch mutation since that can cause issues

**Pull request merged**